### PR TITLE
Portuguese-BR translation update

### PR DESCRIPTION
**Pull request by altiereslima**

### CODE REVIEW

**Comment by lightningterror** [comment]
You could have updated the old branch without creating a new PR but it's too late now.
Git Shell is awesome for that.

File: locales/pt_BR/pcsx2_Main.po [hunk 1]
```diff
@@ -1072,8 +1068,7 @@ msgid "Create a new memory card"
 msgstr "Criar um novo cartão de memória"
 
 #: pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp:59
-#: pcsx2/gui/Panels/DirPickerPanel.cpp:73
-#: pcsx2/gui/Panels/DirPickerPanel.cpp:271
+#: pcsx2/gui/Panels/DirPickerPanel.cpp:73 pcsx2/gui/Panels/DirPickerPanel.cpp:271
```
Comment (lightningterror) [inline locales/pt_BR/pcsx2_Main.po]: Why did you change this part ? It was fine before.

Comment (altiereslima) [reply locales/pt_BR/pcsx2_Main.po]: Poedit does this.

**Comment by altiereslima** [comment]
I already finished

**Comment by lightningterror** [comment]
Can you do a rebase again?
